ivy/src/main/scala/sbt/IvyInterface.scala

```diff
@@ -21,9 +22,9 @@ final case class ModuleInfo(nameFormal: String, description: String = "", homepa
 }
 
 /** Basic SCM information for a project module */
-final case class ScmInfo(browseUrl: URL, connection: String, devConnection: Option[String] = None)
+final case class ScmInfo(browseUrl: URL, connection: String, devConnection: Option[String] = None, tag: Option[String] = None)
 
-final case class Developer(id: String, name: String, email: String, url: URL)
+final case class Developer(id: String, name: String, email: String, url: URL, organization: Option[String] = None, organizationUrl: Option[URL] = None, roles: Seq[String] = Nil, timezone: Option[TimeZone] = None, properties: Map[String, String] = Map.empty)
```

Comment:
Wow, I didn't realize this existed.   Again, the same issue with binary compatibility.

Let me chat with @eed3si9n to see if we come up with a clever way to add these without breaking BC.